docker-tests/it-batch-index/cluster.sh

```diff
@@ -0,0 +1,63 @@
+#! /bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#--------------------------------------------------------------------
+
+# Starts the test-specific test cluster using Docker compose using
+# versions and other settings gathered when building the images.
+
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
```

Comment:
As I see it, the contents of `cluster.sh` would be the same for every test group. Only the cluster type/config changes. Is it possible to avoid the duplication of the `cluster.sh`?

Reply:
> Copy the `cluster.sh` script from an existing test. Add lines to copy
any test-specific files into the `target/shared` folder.

I see this mentioned in `docs/conversion.md` as something that might prevent this. Just guessing here, but couldn't that be done in some other way, say by putting them in `src/test/resources/shared`?

Reply:
Yes, I've been thinking about how to do this. The two "groups" we have now ended up needing the same setup. I'm waiting to see how a few more groups work out to determine if these are special cases, or if the script really does end up being the same. If the same, we can bump it to the parent directory.